lib/simplecov/configuration.rb

```diff
@@ -71,6 +71,14 @@ def filters
       @filters ||= []
     end
 
+    #
+    # Details covers all measurement types of only lines
```

Comment:
"Details covers all measurement types of only lines" — not clear what it should mean. Also, absence of the dot and starting next line with small letter is confusing.

Reply:
Fixed.